vcr/vcr.go

```diff
@@ -88,12 +94,18 @@ type vcr struct {
 	ambassador      Ambassador
 	network         network.Transactions
 	trustConfig     *trust.Config
+	issuerStore     issuer.Store
```

Comment:
why do we need the issuer's store here? Maybe we initialize it in the VCR instance, but it should have no business storing it for later use?